### PR TITLE
frontend: convert `createbackup` into a functional component

### PR DESCRIPTION
**Pull request by thisconnect**
supersedes https://github.com/digitalbitbox/bitbox-wallet-app/pull/1981

To follow the latest conventions and best practices of React, we'd like to convert our class components into functional ones.

Here, we're converting `createbackup.tsx` into a functional component.

This also fixes the hidden error dialog covered by blocking dialog.

Steps to reproduce:
1) inser microSD
2) manage backups / create backup
3) cancel on device

(also test with empty sdCard)

### CODE REVIEW

**Comment by thisconnect** [comment]
diff to #1981 

```diff
diff --git a/frontends/web/src/routes/device/bitbox02/createbackup.tsx b/frontends/web/src/routes/device/bitbox02/createbackup.tsx
index 11edd5d9d..266be4f38 100644
--- a/frontends/web/src/routes/device/bitbox02/createbackup.tsx
+++ b/frontends/web/src/routes/device/bitbox02/createbackup.tsx
@@ -48,7 +48,7 @@ export const Create = ({ deviceID }: TProps) => {
 
   const maybeCreateBackup = async () => {
     setDisabled(true);
-    const check = await checkBackup(deviceID, true);
+    const check = await checkBackup(deviceID, true).catch(console.error);
     if (check) {
       confirmation(t('backup.create.alreadyExists'), result => {
         if (result) {

```

**Comment by thisconnect** [comment]
rebased

**Review by Beerosagos — APPROVED** [review]
tACK!

While testing with an empty sdcard i got a couple times a writing (where the backup list is usually displayed) saying smth like 'error fetching backups', but could not reproduce it to get more info or take a screenshot. Anyway I don't think that is related to the changes in this PR, so I think that is fine.
